### ISSUE TITLE
Error on test

### ISSUE THREAD

**Issue by bajki** [yagnyam-in/sms-gateway-backend#1]
Hi, im getting error 

Non-resolvable parent POM for in.yagnyam:sms-gateway:0.0.1: Could not find artifact in.yagnyam:proxy-parent:pom:1.0-SNAPSHOT and 'parent.relativePath' points at no local POM @ line 15, column 11

Can you help me with this?

**Comment by chenna-reddy** [comment]
Hey, So you want to send SMS from **your** phone (which will be triggered from the backend)?

**Comment by chenna-reddy** [comment]
parent project is at https://github.com/yagnyam-in/proxy-parent. You can try a local install of the parent project.

**Comment by bajki** [comment]
Yes i want to use backend to send sms.

So what are the steps to make ti running.

Download sms-gateway-backend and proxy-parent in one folder or what.

Can you guide me how to run this.

**Comment by bajki** [comment]
I figured out, i put both folders in one but in pom.xml i added ` <relativePath>../proxy-parent</relativePath>`

**Comment by chenna-reddy** [comment]
As no one was using it, I have recently discontinued the server.
Give me some time, I would restore the server and links, you should be able to use the App.

**Comment by bajki** [comment]
This is the best gateway i found, it should be used!

**Comment by chenna-reddy** [comment]
Everything should be working. Download the app from https://www.sg.yagnyam.in/ and test it.

**Comment by bajki** [comment]
I noticed that if app is running in backgroud it does not send sms.

**Comment by chenna-reddy** [comment]
When you click on the App you should see the requests processed. If not can you try relaunching the app and tell me what happens

**Comment by bajki** [comment]
Yes it work's like that, so that is how it should work? Because it say in notification bar that it runs in background but it doesn't send until you open the app

**Comment by chenna-reddy** [comment]
It listens for [FCM](https://firebase.google.com/docs/cloud-messaging/concept-options) messages and sends SMS in real-time. If for some reason, if it doesn't get these real-time messages, it will send pending messages during start-up. I will check why it won't get messages (might take a while)

**Comment by chenna-reddy** [comment]
Also please check if you have given all permissions (like notifications) for this app.
